### PR TITLE
Javascript: New Webpack Fathead

### PR DESCRIPTION
**Pull request by itai421**
## Description of new Instant Answer, or changes

New Webpack Fathead

## Related Issues and Discussions

[#695](https://github.com/duckduckgo/zeroclickinfo-fathead/issues/695)

## People to notify

@pjhampton @moollaza @bfmags 

## Testing & Review
To be completed by Language Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [x] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)

**Code**
- [ ] Uniformly indented, well commented
- [ ] Fetch.sh and Parse.sh run without errors
- [x] Output contains no blank lines, or multi-line entries
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/webpack_config

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@itai421 this is looking good. Please push your `output.txt` to the Pull Request so it can be auto-installed to Beta and reviewed 👍

File: lib/fathead/webpack_config/output.txt [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
```
Comment (pjhampton) [inline lib/fathead/webpack_config/output.txt]: You can't have blank lines in the output. Please remove these 😄

File: lib/fathead/webpack_config/output.txt [hunk 2]
```diff
@@ -0,0 +1,113 @@
+devServer.watchOptions	A										<section class="prog__container"><p><code>object</code></p><p>Control options related to watching the files.</p><p>webpack uses the file system to get notified of file changes. In some cases this does not work. For example, when using Network File System (NFS). <a href="https://www.vagrantup.com/">Vagrant</a> also has a lot of problems with this. In these cases, use polling:</p><pre><code class="lang-js">watchOptions<span class="token punctuation">:</span> <span class="token punctuation">{</span>\n  poll<span class="token punctuation">:</span> <span class="token boolean">true</span>\n<span class="token punctuation">}</span>\n</code></pre><p>If this is too heavy on the file system, you can change this to an integer to set the interval in milliseconds.</p><p>See <a href="https://webpack.js.org/configuration/watch">WatchOptions</a> for more options.</p></section>	https://webpack.js.org/configuration/dev-server#devserver-watchoptions-
```
Comment (pjhampton) [inline lib/fathead/webpack_config/output.txt]: I think we are also going to have to strip out any `<a>` tags from the descriptions.

File: lib/fathead/webpack_config/output.txt (hunk 1 above)
Comment (itai421) [inline lib/fathead/webpack_config/output.txt]: just pushed a fix

File: lib/fathead/webpack_config/output.txt (hunk 2 above)
Comment (itai421) [inline lib/fathead/webpack_config/output.txt]: what do you mean by that?
I already changed relative `<a>` to reference the full url  (so they link to the webpack site and not duckduckgo). Is there more to it?

Comment (pjhampton) [inline lib/fathead/webpack_config/output.txt]: So I mean `<a href="https://www.vagrantup.com/">Vagrant</a>` should become just `Vagrant`. There shouldn't be links in the description AFAIK

Comment (itai421) [inline lib/fathead/webpack_config/output.txt]: That includes links to other webpack config? such as **WatchOptions** at the end of this abstract?

Comment (pjhampton) [inline lib/fathead/webpack_config/output.txt]: Yeah 😄  we can't have links in the description. See [Point 9 in the docs](https://docs.duckduckhack.com/resources/fathead-overview.html#output-fields). You can put the links in there which will show up on the side, kind of like [this](https://duckduckgo.com/?q=array+bsearch&ia=about).

Comment (pjhampton) [inline lib/fathead/webpack_config/output.txt]: But I don't think this is important right now. As long as the tags are stripped out 👍

**Comment by moollaza** [comment]
@itai421 Thanks! This LGTM :+1:

Excellent Work 💯

**Comment by daxtheduck** [comment]
@itai421 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**